apps/server/src/modules/group/controller/api-test/group.api.spec.ts

```diff
@@ -404,6 +404,34 @@ describe('Group (API)', () => {
 						},
 					]);
 				});
+
+				it('should return groups according to name query', async () => {
+					const { loggedInClient, availableGroupInSchool, adminUser } = await setup();
+
+					const response = await loggedInClient.get().query({ nameQuery: availableGroupInSchool.name });
```

Comment:
Looks a query really like this or is it just the beginning of the name?

Reply:
it could also be the end of the name or some substring that can be found in the namestring. I don't know if query is the perfect word for it, but nameSearchString or something like that is not really helping, though it adds precision of some kind...

Reply:
I meant, you test data. You put in availableGroupInSchool.name - the whole name instead of a substring or something like that. I would prefer here a substring as case

Reply:
then you will love the fact, that i now use the last 3 characters of the group name for the test :)